packages/node/src/async/index.ts

```diff
@@ -0,0 +1,17 @@
+import { NODE_VERSION } from '../nodeVersion';
+import { setDomainAsyncContextStrategy } from './domain';
+import { setHooksAsyncContextStrategy } from './hooks';
+
+/**
+ * Sets the correct async context strategy for Node.js
+ *
+ * Node.js >= 14 uses AsyncLocalStorage
+ * Node.js < 14 uses domains
+ */
+export function setNodeAsyncContextStrategy(): void {
+  if (NODE_VERSION >= 14) {
```

Comment:
```suggestion
  if (NODE_VERSION.major && NODE_VERSION.major >= 14) {
```

Reply:
For some reason I assumed this would already be the major version!